### ISSUE TITLE
2110-40 analysis is unable to identify ST 291

### ISSUE THREAD

**Issue by p5kilo** [ebu/pi-list#151]
Hello,

I'm running a local instance of EBU-LIST in docker desktop and it seems unable to analyze the ancillary data packets in my 2110-40 streams. It correctly identifies payload IDs 0x51/0x01 (Film Codes in VANC space (RP215)) and 0x41/0x01 (Payload Identification, HANC space (S352)), but it identifies 0xE3 (Audio data in HANC space (HDTV) (ST 299-1)) incorrectly as an unidentifiable payload type, despite getting the line number and offset correct from the packet.

A 2110-40 plugin for wireshark has no problem identifying the packet as a ST 291 payload type:

ST 2110_40 Data
    Extended Sequence Number: 0x0051
    Length: 24
    ANC_Count: 1
    00.. .... = (F)ield: unspecified or progressive scan (0x0)
    Packet 1: DID 0xe3, SDID 0x00: Audio data in HANC space (HDTV) (ST 299-1)
        0... .... = (C) or Y: Y:Luma
        .000 0000 1001 .... = Line_Number: 9
        .... 0111 1000 1000 = Horizontal_Offset: 1928
        0... .... = S: StreamNum not used
        .000 0001 = StreamNum: 1
        ..11 1000 11.. .... = DID: 0xe3
        .... 0000 0000 .... = SDID: 0x00: Audio data in HANC space (HDTV) (ST 299-1)
        .... ..00 0010 11.. = Data_Count: 11
        UDW Bytes: 2e 05 80 20 38 02 00 80 20 08 02 00 80 20 0b
        Checksum Word: 0x2f6
        [UDW Array: 05 00 03 00 00 00 00 00 00 00 00]

![Payload_unknown](https://user-images.githubusercontent.com/97574929/209872455-8bf4a52e-92cf-41ff-8d1c-3b5b39d7d31e.png)

**Comment by pedro-alves-ferreira** [comment]
Hello! It is expected that audio is transported in ST2110-3x streams and not in ANC packets. Maybe it's why it is not interpreting those correctly. I would have to check the code but it seems plausible.

**Comment by p5kilo** [comment]
> Hello! It is expected that audio is transported in ST2110-3x streams and not in ANC packets. Maybe it's why it is not interpreting those correctly. I would have to check the code but it seems plausible.

It's still a valid ST 291 payload ID, though. Shouldn't it at least be able to identify it?

**Comment by pedro-alves-ferreira** [comment]
Of course. What I meant is that it was probably an oversight. Maybe someone will step forward to fix it.
